tools/update.sh

```diff
@@ -134,16 +134,9 @@ copy_ios_sdk()
 copy_android_sdk()
 {
     echo "*** Android ***"
-    echo "Copying SalesforceAnalytics library"
-    cp -RL $ANDROID_SDK_FOLDER/libs/SalesforceAnalytics src/android/libs/
-    echo "Copying SalesforceSDK library"
-    cp -RL $ANDROID_SDK_FOLDER/libs/SalesforceSDK src/android/libs/
-    echo "Copying SmartStore library"
-    cp -RL $ANDROID_SDK_FOLDER/libs/SmartStore src/android/libs/
-    echo "Copying MobileSync library"
-    cp -RL $ANDROID_SDK_FOLDER/libs/MobileSync src/android/libs/
-    echo "Copying SalesforceHybrid library"
-    cp -RL $ANDROID_SDK_FOLDER/libs/SalesforceHybrid src/android/libs/
+    echo "Copying SalesforceMobileSDK-Android workspace"
+    mkdir -p src/android/libs/mobile_sdk/
+    cp -R $ANDROID_SDK_FOLDER/* src/android/libs/mobile_sdk/
```

Comment:
Here's the update of all MSDK in the plugin.

Reply:
Could we do some pruning ? E.g. remove tests and sample apps and SalesforceReact.

Reply:
Good idea!

Reply:
There is a catch to that.  MSDK's Gradle workspace includes the sample apps and tests in `settings.gradle.kts` and tests via a variety of source sets in the `build.gradle.kts` of each module.  Thinking of our other projects, MSDK is kind of like the LMR "mono-repo".  It won't support having components removed on the fly, especially as Kotlin DSL lets the workspace define customizations in its `buildSrc` moving forward.

If we do have a need to separate the sample apps and tests or any other content, it might need some additional modernization wizardry.  Do we need do tackle that now or later?